traitsui/extras/demo.py

```diff
@@ -523,17 +549,17 @@ class DemoPath(DemoTreeNodeObject):
     parent = Any()
 
     #: Name of file system path to this package:
-    path = Property()
+    path = Property(depends_on='parent.path,name')
+
+    #: The base URL for links:
+    base_url = Property(depends_on='path')
 
     #: Name of the directory:
     name = Str()
 
     #: UI form of the 'name':
     nice_name = Property()
 
-    #: Description of the contents of the directory:
-    description = Property(HTML, depends_on="_description")
-
```

Comment:
Oops, looks like this one needs to stay actually. `DemoPath` does not subclass `DemoFileBase` and the description there is not a property.

Reply:
Shifted to above, but fixed.